test/integration/targets/alternatives/tasks/tests.yml

```diff
@@ -0,0 +1,14 @@
+- block:
+  - include_tasks: remove_links.yml
+  - include_tasks: setup_test.yml
+  # at least two iterations:
+  # - first will use 'link currently absent',
+  # - second will receive 'link currently points to'
+  - include_tasks: test.yml
+    with_sequence: start=1 end=2
+  vars:
+    with_link: '{{ test_conf[0] }}'
+    with_alternatives: '{{ test_conf[1] }}'
```

Comment:
CI was successful because tests weren't run: `posix/ci/group{1,2,3}` was missing in `aliases`.